### PR TITLE
Creating constants for the stats names. 

### PR DESCRIPTION
**Pull request by JeanetteBruno**
Creating one location for  finding the list of stats being collected.  Let me know if you want me to merge this PR and create one more PR with the rest of the changes, or merge the rest of the changes into this PR.

### CODE REVIEW

File: common/stats/stats_names.go [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+	The number of jobs in the inProgress list at the end of each time through the
+	scheduler's job handling loop
+	 */
+
+	SchedAcceptedJobsGauge="schedAcceptedJobsGauge"
+
+	/*
+	the number of tasks that have finished (including those that have been killed)
+	 */
+	SchedCompletedTaskCounter = "completedTaskCounter"
```
Comment (jasonbs10) [inline common/stats/stats_names.go]: Do we want to remove the 'sched' prefix from all of these strings? It should already have 'scheduler' and whatever reason I had to include 'sched' prefix may no longer apply.

File: common/stats/stats_names.go [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+	*/
+	SchedInProgressTasksGauge="schedInProgressTasksGauge"
+
+	/*
+	the number of job requests that have been put on the addJobChannel
+	 */
+	SchedJobsCounter="schedJobsCounter"
+
+	/* TODO - remove? (discuss usefulness)
+	the amount of time it takes to verify a job definition, it to the add job channel and
```
Comment (jasonbs10) [inline common/stats/stats_names.go]: missing word

File: common/stats/stats_names.go [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+	/*
+	the number of times the platform retried sending an end saga message
+	 */
+	SchedRetriedEndSagaCounter= "schedRetriedEndSagaCounter"
+
+	/*
+	The number of jobs waiting to start in the inProgress list at the end of each time through the
+	scheduler's job handling loop.  (No tasks in this job have been started.)
+	 */
+	SchedWaitingJobsGauge="schedWaitingJobsGauge"
```
Comment (dgassaway) [inline common/stats/stats_names.go]: Nit about inconsistent spacing in these var assignments, unless gofmt did that

File: common/stats/stats_names.go (hunk 2 above)
Comment (JeanetteBruno) [inline common/stats/stats_names.go]: done

File: common/stats/stats_names.go (hunk 1 above)
Comment (JeanetteBruno) [inline common/stats/stats_names.go]: I would like to leave the strings exactly as is right now.  Then as I update the graphs, I can update the strings.

File: common/stats/stats_names.go (hunk 3 above)
Comment (JeanetteBruno) [inline common/stats/stats_names.go]: no - I hadn't run fmt yet - will do so with the bulk changes.

**Pull request merged**